pt/lessons/libraries/benchee.md

```diff
@@ -111,27 +113,31 @@ Benchee.run(%{"example function" => fn -> "hi!" end}, [
     comparison: true,
     unit_scaling: :best
   ]
-])
+)
 ```
 
 As opções disponíveis são as seguintes (também documentadas em [hexdocs](https://hexdocs.pm/benchee/Benchee.Configuration.html#init/1)).
 
 * **warmup** - o tempo em segundos para o qual um cenário de _benchmarking_ deve ser executado sem tempos de medição antes do início das medidas reais. Isso simula um sistema de funcionamento "quente". Padrão é 2.
 * **time** - o tempo em segundos por quanto tempo cada cenário de _benchmarking_ individual deve ser executado e medido. Padrão é 5.
-* **inputs** - um mapa com _strings_ que representam o nome de entrada como as chaves e a entrada real como valores. Padrão é `nil`. Vamos abordá-lo em detalhes na próxima seção.
+* **memory_time** - o tempo em segundos de quanto tempo o consumo de memória deve ser medido para cada cenário do benchmarking. Veremos isso mais tarde. O padrão é 0.
+* **inputs** - um mapa com strings que representam o nome da entrada como as chaves e a entrada real como os valores. Também pode ser uma lista de tuplas no formato `{input_name, actual_value}`. O padrão é `nil` (sem entradas). Vamos cobrir isso em detalhes na próxima seção.
 * **parallel** - o número de processos para usar no _benchmark_ de suas funções. Então, se você definir `parallel: 4`, serão gerados 4 processos que executam a mesma função para determinado `time`. Quando estes terminam, então 4 novos processos serão gerados para a próxima função. Isso lhe dá mais dados no mesmo tempo, mas também adiciona mais carga ao sistema interferindo nos resultados do _benchmark_. Isso pode ser útil para simular um sistema sobrecarregado, o que algumas vezes é útil, mas deve ser usado com algum cuidado pois isso pode afetar os resultados de maneiras imprevisíveis. Padrão é 1 (o que significa nenhuma execução em paralelo).
-* **formatters** - uma lista de funções do formatador que você deseja executar para gerar os resultados de benchmarking do conjunto ao usar `Benchee.run/2`. Funções precisam aceitar um argumento (que é a suite de _benchmarking_ para todos dados) e então usá-la para produzir a saída. Padrão é o formatador de console embutido chamando `Benchee.Formatters.Console.output/1`. Vamos abordar mais sobre isso em uma seção posterior.
+* **formatters** - uma lista de formatadores é um módulo implementando o comportamento do formatador, uma tupla do referido módulo e opções que ele deve tomar ou funções formatadoras. Eles são executados ao usar `Benchee.run/2`. Funções precisam aceitar um argumento (que é o conjunto de benchmarking com todos os dados) e então usar isso para produzir a saída. O padrão é o formatador de console embutido no `Benchee.Formatters.Console`. Nós cobriremos isso mais em uma seção posterior.
```

Comment:
```suggestion
* **formatters** - uma lista de formatadores podendo ser um módulo implementando o comportamento do formatador, uma tupla do referido módulo e opções que ele deve tomar ou funções formatadoras. Eles são executados ao usar `Benchee.run/2`. Funções precisam aceitar um argumento (que é o conjunto de benchmarking com todos os dados) e então usar isso para produzir a saída. O padrão é o formatador de console embutido no `Benchee.Formatters.Console`. Nós cobriremos isso mais em uma seção posterior.
```

@herminiotorres I think here we could avoid the literal translation in order to have a sentence more easy to understand. WDYT?